nautobot/core/ui/object_detail.py

```diff
@@ -219,47 +299,73 @@ class _ObjectDetailAdvancedTab(Tab):
     tab_id = "advanced"
     tab_label = "Advanced"
 
-    def __init__(self):
-        super().__init__(
-            layouts=(
-                LayoutTwoColumn(
-                    left_panels=(
-                        # TODO
-                        ObjectFieldsPanel(
-                            label="Object Details",
-                            fields=["id", "natural_slug", "slug"],
-                            ignore_nonexistent_fields=True,
-                        ),
-                        ObjectFieldsPanel(
-                            label="Data Provenance",
-                            fields=["created", "last_updated"],
-                            ignore_nonexistent_fields=True,
-                            # TODO add created_by/last_updated_by derived fields and link to REST API
-                        ),
-                        # TODO add advanced_ui custom fields and relationships
-                    ),
+    def __init__(self, **kwargs):
+        kwargs.setdefault("weight", 1000)
+        kwargs.setdefault(
+            "panels",
+            [
+                ObjectFieldsPanel(
+                    label="Object Details",
+                    section=SectionChoices.LEFT_HALF,
+                    weight=100,
+                    fields=["id", "natural_slug", "slug"],
+                    ignore_nonexistent_fields=True,
                 ),
-                LayoutFullWidth(),
-            )
+                ObjectFieldsPanel(
+                    label="Data Provenance",
+                    section=SectionChoices.LEFT_HALF,
+                    weight=200,
+                    fields=["created", "last_updated"],
+                    ignore_nonexistent_fields=True,
+                    # TODO add created_by/last_updated_by derived fields and link to REST API
+                ),
+                # TODO add advanced_ui custom fields and relationships
+            ],
         )
+        super().__init__(**kwargs)
 
 
 class _ObjectDetailContactsTab(Tab):
     """Built-in class for a Tab displaying information about contact/team associations."""
 
     tab_id = "contacts"
 
-    def __init__(self):
-        super().__init__(
-            layouts=(
-                LayoutFullWidth(
-                    panels=(
-                        # TODO
-                        TemplatePanel(label="Contact Associations", template_string="Contact Associations Table"),
-                    ),
+    def __init__(self, **kwargs):
+        kwargs.setdefault("weight", 1100)
```

Comment:
I am a little confused as to why we are starting from 0 in the ObjectDetailMainTab and jump to 1000 in the ObjectDetailAdvancedTab.